app/components/Profile.jsx

```diff
@@ -0,0 +1,105 @@
+import PersonalInfo from '@/components/PersonalInfo';
+import Address from '@/components/Address';
+import { useState, useEffect } from 'react';
+import ProfileImageWrapper from '@/components/ProfileImageWrapper';
+import PropTypes from 'prop-types';
+
+export default function Profile({ userID = 5 }) {
+  const [isLoading, setIsLoading] = useState(true);
+  const [profileData, setProfileData] = useState({
+    first_name: '',
+    last_name: '',
+    phone_number: '',
+    email: '',
+    country: '',
+    town: '',
+    address: '',
+    image: '',
+  });
+
+  // Handle input changes
+  const handleChange = (event) => {
+    const { name, value } = event.target;
+    setProfileData({
+      ...profileData,
+      [name]: value,
+    });
+  };
+
+  // Fetch user data
+  useEffect(() => {
+    const fetchUserData = async (userID) => {
+      setIsLoading(true);
+      try {
+        const response = await fetch(
+          `http://localhost:3000/api/user/${userID}`,
```

Comment:
With deployments, we can't use URLs this way. It is locally address is `http://localhost:3000` , but in deployment it will be something Vercel has generated. Since they are on the same address as frontend, the easiest would be use `fetch('/api/user/${userID}')`. Other solution -- use env variable